includes/editor-templates/global.php

```diff
@@ -51,7 +72,7 @@
 			</ul>
 		</div>
 		<div class="e-view e-con-select-preset">
-			<div class="e-con-select-preset__title"><?php echo esc_html__( 'Select your Structure', 'elementor' ); ?></div>
+			<div class="e-con-select-preset__title"><?php echo esc_html__( 'Select your structure', 'elementor' ); ?></div>
```

Comment:
maybe extract esc_html__( 'Select your structure', 'elementor' ); to a function ?